src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0016Tests.cs

```diff
@@ -1,61 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System.Threading.Tasks;
-using Xunit;
-using Verifier = Azure.ClientSdk.Analyzers.Tests.AzureAnalyzerVerifier<Azure.ClientSdk.Analyzers.OperationConstructorAnalyzer>;
-
-namespace Azure.ClientSdk.Analyzers.Tests
-{
-    public class AZC0016Tests
-    {
-        [Fact]
-        public async Task AZC0016ProducedForOperationTypesWithoutProtectedCtor()
-        {
-            const string code = @"
-namespace RandomNamespace
-{
-    public class Operation { }
-
-    public class [|CertificateOperation|]: Operation
```

Comment:
This is a false positive that @heaths has found. Id here is  AZC0005, not AZC0016